.gitignore

```diff
@@ -16,11 +16,17 @@ x64/
 [Bb]in/
 [Oo]bj/
 
+[Tt]ools/
```

Comment:
I think we need to put the output elsewhere.  Having a directory named "Tools" is far too common.  It actually breaks Roslyn repo because of our existing Tools directory.  I think putting the tools under `Binaries\Tools` is a better location.  That directory is already excluded.